app/assets/javascripts/discourse/templates/preferences-second-factor.hbs

```diff
@@ -143,7 +143,8 @@
             type="submit"
```

Comment:
seems like `type="submit"` was already there ?

Reply:
You're right, I removed the duplicate. It is still working fine for me